### DISCUSSION TITLE
sticky symbol position

### DISCUSSION THREAD

**Discussion by erifranck** [keiwando/toonsquid#161] [Ideas]
Basically when I was making a symbol to use in a lipsync I found an issue in the workflow, because if I need to repeat one position I have to duplicate same frame, could be easier if I can select loop tipe fixed and when I select the frame in the symbol slider it should be sticky and manage that with keyframes then when I select a new position create a new keyframe and I don't have to duplicate symbols frames.

**Comment by Cat-Butler** [comment]
Hi @erifranck - this is already available, but there a couple of conditions you need to meet.

if your mouth symbol has only **single-frame** mouth alternatives (just one frame for each mouth shape), and each has its own Frame Marker (descriptive label associated with a specific frame), then each Time keyframe in the main timeline will automatically have a **Hold** easing curve.

This gives you the "sticky" (Hold) behaviour you are after, when you select a mouth shape from the options in your symbol frames using a Time keyframe. 
In addition, it also lets you choose from a visual array of mouth shapes (Frame Markers).

Of course, this isn't limited to mouth symbols, it will work with hand positions, eyes, different wing positions on a flying bird, on/off or colour states for notification lights on equipment, etc.
